src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -289,14 +289,24 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         if (issuanceMintAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Mint tokens to address
-        _mint(_receiver, issuanceMintAmount);
+
+        // Use abstract function to distribute issuance tokens
+        _distributeIssuanceToken(_receiver, issuanceMintAmount);
+
         // Emit event
         emit TokensBought(
             _receiver, _depositAmount, issuanceMintAmount, _msgSender()
         );
     }
 
+    /// @dev    Abstract function to distribute issuance tokens.
```

Comment:
Should be a `@notice`. Not developer-specific.

Reply:
Also `Abstract` is not correct I believe, as noted above.

Reply:
Done